coding-for-reusability/Exercise5/solution/Solution.tsx

```diff
@@ -29,7 +29,7 @@ type ApiResponse<Data extends Entity> =
       error: string;
     };
 
-async function fetchMock<Data extends Entity>(
+async function mockResponse<Data extends Entity>(
```

Comment:
This is a very small suggestion… you could either not have the `return new Promise` syntax because this is an `async` function, or remove the `async` from this declaration because it always returns a Promise. 🤷‍♂️